k8s_helm_charts/zdm/templates/sts.yaml

```diff
@@ -0,0 +1,134 @@
+# pull in these templates outside of range loop
+{{ $zdm_fullname := include "zdm.fullname" . -}}
+{{- $zdm_labels := include "zdm.labels" . -}}
+{{- $zdm_selectorLabels := include "zdm.selectorLabels" . -}}
+
+# calculate a variable that contains all proxy service addresses
+{{ $service_addresses := "" -}}
+{{- range $index := until (.Values.proxy.count | int) -}}
+    {{- $service_addresses = printf "%s,$(%s_%s_SERVICE_HOST)" $service_addresses ($zdm_fullname | upper | replace "-" "_") ($index | toString) -}}
+{{- end -}}
+{{- $service_addresses = $service_addresses | trimPrefix "," -}}
+
```

Comment:
While it isn't specific to this PR, I recommend that you document every property in values.yaml. Think of as your user-facing, client API :)

Reply:
Good recommendation. I've now completely revised values.yaml based on the best practices described in [this helm documentation page](https://helm.sh/docs/chart_best_practices/values/).